resources/assets/js/slideshow.js

```diff
@@ -246,6 +247,16 @@ function handleControlClick(event) {
 
 function createAnnotations(annotations) {
 	let annotationsHtml = '';
+	const badge = (profile) => {
+		if (profile.roles.indexOf('moderator') !== -1) {
+			return `
+				<div class="annotation-meta-signature">
```

Comment:
Let's remember to update it to 
```suggestion
				<div class="comment-meta-signature">
```
if https://github.com/bethinkpl/wnl-platform/pull/1455 gets merged first.